airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -318,7 +318,7 @@ public JdbcDatabase createDatabase(final JsonNode sourceConfig) throws SQLExcept
         driverClassName,
         jdbcConfig.get(JdbcUtils.JDBC_URL_KEY).asText(),
         connectionProperties,
-        getConnectionTimeout(connectionProperties, driverClassName));
```

Comment:
This change concerns me a little bit. There's some special logic for timeouts WRT postgres, mysql or sqlserver. I think we should move `getConnectionTimeout(Map<String, String>, String?` out of the companion object instead of using the method that doesn't take a `driverClassName`

Reply:
I marked the function `getConnectionTimeout(Map<String, String>, String?` in the companion object in `JdbcConnector` as `@JvmStatic`. I'm not really sure why this class exists in the whole hierarchy. Feels like it can be guttedbut this seems to be the most innocuous change IMO

Reply:
that works. Thank you.
The class should eventually be in the hierarchy, and the DB specific behavior should be decided by each connector, rather than being coded inside the CDK, which means it'll eventually move back to being non-static